### DISCUSSION TITLE
Notification Policies based on Prometheus Alerts

### DISCUSSION THREAD

**Discussion by htrmeira** [grafana/grafana#43067] [Alerting]
Hi team,

# Use Cases

* Get alerted based on Prometheus alerts: For those who already have a Prometheus in place with its own alerts configured, it would be nice to be able to configure notification policies based on this alerts.

# General Information

Today we can create notification policies based on alerts created in Grafana itself and the new Grafana Alerts Dashboard already shows Prometheus alerts after configuring the data source. But the notification policies are not applied to these alerts.

(Please, let me know if there is a template for these requests out there and thank you very much in advance, any feedback is welcome).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
